ghcide/test/data/import-placement/PragmaNotAtTopMultipleComments.hs

```diff
@@ -0,0 +1,22 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+{-# OPTIONS_GHC -Wall,
+   OPTIONS_GHC -Wno-unrecognised-pragmas #-}
```

Comment:
The expected version of this test doesn't have the `OPTIONS_GHC -Wno-unrecognized-pragmas` line. I think the expected version of the test should have this line unless the test does more than add an import.

Reply:
no you're right, that's updated now.